components/JournalEntryList/index.js

```diff
@@ -0,0 +1,48 @@
+import { titles } from "/lib/data.js";
+import styled from "styled-components";
+import { v4 as uuidv4 } from "uuid";
+
+export default function JournalEntryList() {
+  return (
+    <>
+      <header>
+        <Heading>Your Entries</Heading>
+      </header>
+      <main>
+        <EntryList>
+          {titles.map((title) => (
+            <ListItem key={uuidv4()}>
```

Comment:
Here you create a new key for each list entry at each rendering. With this, all the performance advantages that react offers us vanish into thin air.

A better way is to attach an id to your titles dataset and to use that instead.

```suggestion
            <ListItem key={title.id}>
```